src/test/K6/src/tests/platform/storage/appowner/completeconfirmation.js

```diff
@@ -0,0 +1,70 @@
+/* Pre-reqisite for test: 
+    1. MaskinPorteTokenGenerator https://github.com/Altinn/MaskinportenTokenGenerator built
+    2. Installed appOwner certificate
+    3. Send maskinporten token as environment variable after generating the token
+
+    This test script sets complete confirmation as app owner on all the hard deleted instances of an app.
+    example: k6 run -i 20 -u 10 /src/tests/platform/storage/appowner/completeconfirmation.js -e env=test -e org=ttd 
+    -e level2app=rf-0002 -e subskey=*** -e maskinporten=token
+*/
+
+import { check } from "k6";
+import { addErrorCount, printResponseToConsole } from "../../../../errorcounter.js";
+import * as storageInstances from "../../../../api/storage/instances.js"
+import { convertMaskinPortenToken } from "../../../../api/platform/authentication.js"
+import * as setUpData from "../../../../setup.js";
+
+const appOwner = __ENV.org;
+const level2App = __ENV.level2app;
+const maskinPortenToken = __ENV.maskinporten;
+const createdDateTime = __ENV.createddate;
+
+export const options = {
+    thresholds: {
+        "errors": ["count<1"]
+    },
+    setupTimeout: '5m'
+};
+
+//Function to authenticate a app owner, get all archived hardeleted and not complete confirmed instances of an app and return data
+export function setup() {
+    var altinnStudioRuntimeToken = convertMaskinPortenToken(maskinPortenToken, "true");
+    var data = {};
+    data.runtimeToken = altinnStudioRuntimeToken;
+    var hardDeletedAppInstances = storageInstances.findAllHardDeletedInstances(altinnStudioRuntimeToken, appOwner, level2App, createdDateTime);
```

Comment:
this is an expensive operation as we will use R/Us from cosmos.. Can we get these instance Ids any other way ?

Reply:
if this is the logic you are planning to use for load testing that is.. We should store the id from when you originally interact with the instances to avoid additional traffic for cosmos

Reply:
Requests are sent to storage API to find all archived instances which are created after a specific date. Hope a set of calls to storage to find archived instances will not take R/Us.
After that, the filtering of hard deleted and not completed confirmed instances is done within k6 code (no additional requests are sent to storage api)

this script is not to be used for performance test but to run after a performance test is run.

Reply:
now the test takes instance details from a external csv file